.github/workflows/deploy-to-environment.yml

```diff
@@ -154,14 +154,17 @@ jobs:
 
       - name: Deploy Backend to Environment
         run: |
+          DFX_DEPLOY_FLAGS=() # Arguments to be added to the dfx deploy command.
+          [[ "${{ github.event.inputs.force-backend }}" != "true" ]] || DFX_DEPLOY_FLAGS+=('--yes' '--upgrade-unchanged') # Corresponds to: dfx deploy --yes --upgrade-unchanged
```

Comment:
`!= "true"` is correct here? it's not the contrary?

Reply:
It is correct.  If true, more flags are added.  If undefined, false or any other value, no flags are added.

Reply:
Gotcha.